pkg/aws/appmesh.go

```diff
@@ -231,6 +246,11 @@ func (v *VirtualNode) BackendsSet() set.Set {
 
 // GetVirtualNode calls describe virtual node.
 func (c *Cloud) GetVirtualNode(ctx context.Context, name string, meshName string) (*VirtualNode, error) {
+	begin := time.Now()
+	defer func() {
+		c.stats.SetRequestDuration("virtual_node", name, "get", time.Since(begin))
```

Comment:
is it better to normalize? `<meshName>.<virtualNodeName>`

Reply:
I would not use dots in Prometheus metrics, we could add the mesh a separate label if you want.

Reply:
Agreed with @stefanprodan, let's add as a separate label.

Reply:
I would like to do this and add more instrumentation in a followup PR. The metrics as they are now are sufficient for the Grafana dashboard I made in https://github.com/aws/eks-charts/pull/30